.vortex/installer/src/Prompts/Handlers/Modules.php

```
@@ -115,6 +115,14 @@ public function process(): void {
       }
     }
 
+    if ($removed_packages !== []) {
+      JsonManipulator::updateFile($t . '/composer.json', function (JsonManipulator $cj) use ($removed_packages): void {
+        foreach ($removed_packages as $removed_package) {
+          $cj->removeSubNode('require', $removed_package);
+        }
+      });
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\DrevOps\VortexInstaller\Utils\JsonManipulator' in method 'process'. (undefined)

(StaticAccess)